packages/inspector/src/components/EntityInspector/ScriptInspector/parser.ts

```diff
@@ -0,0 +1,222 @@
+import { parse } from '@babel/parser';
+import type {
+  Identifier,
+  TSTypeAnnotation,
+  Expression,
+  FunctionParameter,
+  ClassMethod,
+  TSParameterProperty,
+} from '@babel/types';
+import { engine } from '@dcl/ecs';
+
+import type { ScriptParamUnion } from './types';
+
+function getValueAndTypeFromExpression(expression: Expression): ScriptParamUnion {
+  switch (expression.type) {
+    case 'NumericLiteral':
+      return { type: 'number', value: expression.value };
+    case 'BooleanLiteral':
+      return { type: 'boolean', value: expression.value };
+    case 'StringLiteral':
```

Comment:
should we use the StringLiteral as the `default` too?
```suggestion
    case 'StringLiteral':
    default:
```